packages/polymath-shared/babel.config.js

```diff
@@ -0,0 +1,7 @@
+const base = require('../../config/babel.config.js');
+
+module.exports = {
+  babelrcRoots: ['../packages/*'],
```

Comment:
I think this line is useless/wrongly written as it should be `['../*']` or I'm missing something?

Reply:
Good catch, this us outdated code 💀